.release-version

```diff
@@ -1 +1 @@
-1.10.1
```

Comment:
I would say this should be a minor version update, rather than a patch update - so go to 1.11.0. Since we're adding a whole table?